pkg/costmodel/aggregation.go

```diff
@@ -2275,15 +2275,15 @@ func (a *Accesses) ComputeAllocationHandler(w http.ResponseWriter, r *http.Reque
 	// Otherwise it is computed at the cluster level. (Not relevant if idle
 	// is not included.)
 	idleByNode := qp.GetBool("idleByNode", false)
-	sharedLoadBalancer := qp.GetBool("sharelb", false)
+	// sharedLoadBalancer := qp.GetBool("sharelb", false)
```

Comment:
what's going on here? Why are we removing the shared loadbalancer functionality for a special case?

Reply:
Ah, thanks for the catch here--I believe that had been broken when this was first worked on, and got disabled back then so that it would run

Reply:
Has that been fixed and these changes can be reverted?